backend/test/richard_burton_web/controllers/publication_controller_test.exs

```diff
@@ -623,4 +629,44 @@ defmodule RichardBurtonWeb.PublicationControllerTest do
       assert expected_data == response(conn, 200)
     end
   end
+
+  describe "update" do
+    def setup_update(%{conn: conn}) do
+      {:ok, publication} = @publication_attrs |> create_publications()
+      path = conn |> publication_path(:update, publication.id)
+
+      %{publication: publication, path: path}
+    end
+
+    # TODO:
+    #
+    # - tests de errores de validación
+    # - fallback controller para 404
+    #
```

Comment:
Would also be nice to add test to ensure the fingerprints are updated along with the parameters.